triaging.rst

```diff
@@ -36,6 +36,12 @@ Responsibilities include:
     - Skip issue
     - Good first issue
     - Other categorizations
+    
+ .. note::
+ 
+   Triagers should only close PRs that could be considered spam. They can
+   suggest for other PRs to be closed, but core developers are responsible for
+   making the final decision.
```

Comment:
Maybe mention something about the use of labels to note other core devs to have a look?

Based on @brettcannon comment
> We could add a label for triagers to mark a PR as potentially stale so it gets flagged up to core devs to have a look.

Reply:
>Maybe mention something about the use of labels to note other core devs to have a look?

I'm definitely on board with adding the label he was talking about, but we'll have to wait for that to be implemented before adding it to the devguide.

Reply:
Ah yes.  :-)

Reply:
@eamanu Since Mariatta added the label, I updated the section accordingly and added a description for the label.